src/components/core/geoChart/GeoChartRenderer.tsx

```diff
@@ -114,6 +117,8 @@ export default class GeoChartRenderer extends React.PureComponent<IGeoChartRende
         }
 
         const geoData = getGeoData(buckets, executionResponse.dimensions);
+        this.geoData = geoData;
```

Comment:
it could be just `this.geoData = getGeoData(buckets, executionResponse.dimensions);`
and `this.geoData` for below needs

Reply:
updated :)